src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exceptions.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection dbConnection = ConnectionUtil.getConnection();
+                PreparedStatement statement = dbConnection
+                        .prepareStatement("INSERT INTO manufacturers (name,country, is_deleted)"
+                                        + " VALUES(?,?,?)",
+                        Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
```suggestion
    public Manufacturer create(Manufacturer manufacturer) {
        String query = "INSERT INTO manufacturers (name, country)"
                                        + " VALUES(?,?)"
        try (Connection dbConnection = ConnectionUtil.getConnection();
                PreparedStatement statement = dbConnection
                        .prepareStatement(query,Statement.RETURN_GENERATED_KEYS)) {
```
lets move select query into local variable at the beginning of method for better readability
same for other methods
is_deleted should be set to default false in db. no need to manually do it